program/src/lib.rs

```diff
@@ -512,7 +517,6 @@ pub mod syscalls;
 pub mod system_instruction;
 pub mod system_program;
 pub mod sysvar;
-pub mod wasm;
```

Comment:
We shouldn't need to make this breaking change -- the wasm code can still live in `solana-program`, to be used by downstream builds.